efficiency/misc/make_lstpage.py

```diff
@@ -0,0 +1,344 @@
+#!/bin/env python
+
+import os
+import glob
+import sys
+
+#___________________________________________________________________________________________________
+def write_pages_v2(directory, objecttypes):
```

Comment:
Same comment as before: any usage for `v1`?

Reply:
i can  nuke them..